sql/pgsql/_debug_api.sql

```
@@ -113,7 +122,8 @@ ORDER BY "PickemGroupId", "ContestId", rn;
 
 
 
-select * from public."ContestPrediction"
+select * from public."ContestPrediction" limit 10;
```

SQLFluff (4.3.0)

[error] Too many consecutive blank lines.

(LT15)